src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -207,14 +178,13 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     }
 }
 
-
 //! Clears the toolbox.
 void medRegistrationSelectorToolBox::clear(void)
```

Comment:
```
medPipelineDataDropWidget::endDrop()
medPipelineInstance::endStep
```
aaaaand it seems to be ok. Under QtCreator, open `medAbstractWorkspace`, right click on clear() and "Trouver les utilisations" (or something like that in english for instance)